src/map/skill.cpp

```diff
@@ -2183,8 +2183,6 @@ int skill_additional_effect( struct block_list* src, struct block_list *bl, uint
 		[[fallthrough]];
 	case WM_METALICSOUND:
 	case WM_REVERBERATION:
-	case TR_RHYTHMSHOOTING:
```

Comment:
Not mentioned in the changelog?
If you are sure about this, just ignore this comment.

Reply:
This is not mentioned in the official changelog (like `TR_METALIC_FURY`). But the skill description has been changed as well